files/KoreBuild/scripts/commands/Commands.psm1

```diff
@@ -0,0 +1,8 @@
+#requires -version 4
+
+Set-StrictMode -Version 2
+$ErrorActionPreference = 'Stop'
+
+. "$PSScriptRoot\Invoke-DockerBuild.ps1"
```

Comment:
Is there some way that I'll only have to include either these `. "$script"` imports or the `FunctionsToExport` stuff in the .psd1?